proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -105,5 +105,11 @@ message Webhook {
   // The webhook repository owner name.
   string owner_name = 6;
   // The subscriber's callback URL where notifications are delivered.
+  // Currently we only support Connect-powered backends with application/protobuf as the content type.
+  // Make sure that your URL ends with `/buf.alpha.webhook.v1alpha1.EventService/Event`.
+  // For more information about connect see `https://connect.build`.
+  //
+  // Other delivery mechanisms (like JSON or other protos) may be added in the
+  // future.
```

Comment:
>Other delivery mechanisms (like JSON or other protos)

This doesn't make much sense. Even with connect we would have to pick wether it is sent as proto or json.

The URL doesn't strictly specify encoding, `buf.alpha.webhook.v1alpha1.EventService/Event` could be called with either json or proto payloads.

Reply:
Removing

Reply:
It's probably not worth calling out. We can explain this in the future when we revisit that decision